socialhome/streams/app/components/StreamElement.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <div>
+        <div v-html="htmlSafe"></div>
+        <author-bar
+            v-if="showAuthorBar"
+            v-bind="author"
+            :is-user-author="isUserAuthor"
+            :is-user-authentificated="isUserAuthentificated"
+        ></author-bar>
+        <div class="grid-item-bar">
+            <div class="row">
+                <div class="col-9">
+                    <!-- TODO: Modal and Socket need a rewrite to get rid of grid-item-open-action -->
+                    <span
+                        v-if="edited"
```

Comment:
This `v-if` should be for the `{{ editedText }}` not for the whole timestamp. Now unless it is edited the whole timestamp is hidden.